app/i18n/en/gen.php

```diff
@@ -156,6 +156,7 @@
 		'damn' => 'Damn!',
 		'default_category' => 'Uncategorized',
 		'no' => 'No',
+		'not_applicable' => 'N/A',
```

Comment:
I was thinking of "N/A" because people are used to it. Like when using Excel.
Anyway, it's easy to change if needed!